aerolyzer/horizon.py

```diff
@@ -1,45 +1,35 @@
-import sys
 import cv2
-import os
+import sys
 import numpy as np
-from matplotlib import pyplot as plt
 
-def test():
-	print "hi"
-def _testAnalyze(self, directory):
-	included_extensions = ['jpg','JPG']
-	files = [fn for fn in os.listdir(directory) if any(fn.endswith(ext) for ext in included_extensions)]
-	for image in files:
-		img_noblur = cv2.imread(str(directory)+image)
-		img_noblur = cv2.cvtColor(img_noblur, cv2.COLOR_BGR2GRAY)
-		self.getHorizon(str(directory), img_noblur,image)
+	
+def Horizon(directory, image, output=0):
+	imgOg = cv2.imread(str(directory)+image) # Read image
+	
+	img_YUV = cv2.cvtColor(imgOg,cv2.COLOR_BGR2YCR_CB)	# Convert from BGR to YCRCB
+	b, r, g = cv2.split(img_YUV) 						# Split into blue-green-red channels
+	imgBlueEqu = cv2.merge((cv2.equalizeHist(b), r, g))	# Equalize Blue Channel & Merge channels back
+	img_BGR = cv2.cvtColor(imgOg,cv2.COLOR_YCR_CB2BGR)	# Convert from YCRCB to BGR
+
+	blur = cv2.GaussianBlur(img_BGR,(5,5),0)			# Perform a gaussian blur
+
+	img_GREY = cv2.cvtColor(blur,cv2.COLOR_BGR2GRAY)	# Convert to Greyscale
+
+	ret, thresh = cv2.threshold(img_GREY,120,125,cv2.THRESH_BINARY)	# Threshold image to segment sky
```

Comment:
How were the values for the threshold chosen?

Reply:
Values for threshold were chosen through ad hoc guess and check.